instrumentation/action_mailer/test/opentelemetry/instrumentation/action_mailer/instrumentation_test.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+describe OpenTelemetry::Instrumentation::ActionMailer do
+  let(:instrumentation) { OpenTelemetry::Instrumentation::ActionMailer::Instrumentation.instance }
+
+  it 'has #name' do
+    _(instrumentation.name).must_equal 'OpenTelemetry::Instrumentation::ActionMailer'
+  end
+
+  it 'has #version' do
+    _(instrumentation.version).wont_be_nil
+    _(instrumentation.version).wont_be_empty
+  end
+
+  describe '#install' do
```

Comment:
Would you be open to adding some tests for the logic in `resolve_email_address`, `ecs_mail_convention`, and `transform_payload`? Since they're private methods, I know that's not always a common practice, but I'm concerned about not having coverage for their logic.

Reply:
Yes, I will updated the test case

Reply:
Thank you!